WordPress/Classes/ViewRelated/NUX/LoginPrologueViewController.swift

```diff
@@ -55,7 +55,15 @@ class LoginPrologueViewController: UIViewController {
 
     @IBAction func signupTapped() {
         if Feature.enabled(.socialSignup) {
-            performSegue(withIdentifier: "showSigninV2", sender: self)
+
```

Comment:
This could be done with a storyboard reference instead. That sound ok?

Reply:
Hey @nheagy . When I added a storyboard reference for Signup, I found an interesting issue. The storyboard reference is triggered before the Signup Button action. This results in the new Signup storyboard *always* being displayed, regardless of the Feature Flag setting, because it is called *before* we catch the button action in `signupTapped` and decide which segue to perform. Since this is temporary anyway, I'm going to leave it as-is. In the end, using a Signup storyboard reference will work fine since it will be launched from the new Pick Auth Method screen and won't conflict with the current flow.

Reply:
Ah, that sounds great.